website/versioned_docs/version-0.76/getting-started.md

```diff
@@ -0,0 +1,140 @@
+  - Select the `Debug` configuration and the `x64` platform from the combo box controls to the left of the `Run` button and underneath the `Team` and `Tools` menu item.
+  - Run `yarn start` (or `npm start`) from your project directory, and wait for the React Native packager to report success.
+  - Click the `Run` button to the right of the platform combo box control in VS, or select the `Debug`->`Start without Debugging` menu item. You now see your new app and Chrome should have loaded `http://localhost:8081/debugger-ui/` in a new tab. Press `F12` or `Ctrl+Shift+I` in Chrome to open its Developer Tools. :tada:
+
+- With VS Code
+  - Open your applications folder in VS Code.
+  - Install the [React Native Tools](https://marketplace.visualstudio.com/items?itemName=msjsdiag.vscode-react-native) plugin for VS Code.
+  - Create a new file in the applications root directory, `.vscode/launch.json` and paste the following configuration:
+  ```json
+  {
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "name": "Debug Windows",
+        "cwd": "${workspaceFolder}",
+        "type": "reactnative",
+        "request": "launch",
+        "platform": "windows"
+      }
+    ]
+  }
+  ```
+  - Press `F5` or navigate to the debug menu (alternatively press `Ctrl+Shift+D`) and in the Debug drop-down select "Debug Windows" and press the green arrow to run the application.
+
+## Authoring Native Modules
+
+See [Native Modules and React Native Windows](native-modules.md).
+
+## Building a standalone React Native Windows App
+
+Follow these steps to build a version of your app that you can install or publish to the store. This version will package your bundle and assets into the APPX package so you don't need to run Metro.
+
+- Open the solution in Visual Studio
+- Select the Release configuration from the Configuration Manager drop-down.
+- Build the solution. You can now launch without first launching Metro.
+- If you want to build an APPX package to share or publish, use the **Project** > **Publish** > **Create App Packages...** option.
+
+> The Debug configuration uses the Web Debugger by default, which means the application's JavaScript code runs in Chrome.<br>
+> If you're getting different runtime behavior between the Release and Debug configurations, consider disabling the `UseWebDebugger` setting in [`App.cpp`](https://github.com/microsoft/react-native-windows/blob/6b415659aa017dbc41e3f28e817fb768a8e80435/vnext/template/cpp-app/src/App.cpp#L30) or [`App.xaml.cs`](https://github.com/microsoft/react-native-windows/blob/6b415659aa017dbc41e3f28e817fb768a8e80435/vnext/template/cs-app/src/App.xaml.cs#L20) to get the same behavior in the Debug configuration.
+
+See also this article for additional details: https://techcommunity.microsoft.com/t5/windows-dev-appconsult/getting-started-with-react-native-for-windows/ba-p/912093#
+
+</body>
+</html>
```

Comment:
These HTML tags shouldn't be here. Please remove from here and from `docs/getting-started.md`.
```suggestion
```